modules/1_prepare/prepare.tf

```diff
@@ -449,20 +447,41 @@ resource "null_resource" "rhel83_fix" {
   }
 }
 
-resource "ibm_pi_network_port" "bastion_vip" {
+resource "null_resource" "force_use_utc" {
+  count      = var.force_utc ? local.bastion_count : 0
+  depends_on = [null_resource.rhel83_fix]
+
+  connection {
+    type        = "ssh"
+    user        = var.rhel_username
+    host        = data.ibm_pi_instance_ip.bastion_public_ip[count.index].external_ip
+    private_key = var.private_key
+    agent       = var.ssh_agent
+    timeout     = "${var.connection_timeout}m"
+  }
+  provisioner "remote-exec" {
+    inline = [
+      "sudo timedatectl set-timezone UTC",
```

Comment:
In certain circumstances, the images used are not configured with UTC, however the coreos nodes default to UTC. If Sydney, Chennai or other locations before UTC are used the certificates are evaluated as After when they should be in sync.